mod-spring-petstore/Jenkinsfile

```diff
@@ -0,0 +1,20 @@
+buildMvn {
+  publishModDescriptor = true
+  mvnDeploy = true
+  doKubeDeploy = true
+  buildNode = 'jenkins-agent-java11'
```

Comment:
Should be 'jenkins-agent-java17'
https://dev.folio.org/faqs/how-to-specify-backend-java-ci/

Reply:
Done